tensorboard/webapp/customization/customization_test.ts

```diff
@@ -0,0 +1,113 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {Component, NgModule, Optional} from '@angular/core';
+import {TestBed} from '@angular/core/testing';
+
+import {CustomizationModule} from './customization_module';
+
+/**
+ * Marker class for the component that can be customized in these tests.
+ * Customizations should implement this type and be provided.
+ */
+export class CustomizableComponentType {}
+
+/**
+ * Parent class that uses the <tb-customization> component.
+ */
+@Component({
+  selector: 'parent-component',
+  template: `
+    <tb-customization [customizableComponent]="customizableComponent">
+      <div class="default">Showing Default Text!</div>
```

Comment:
Would you mind altering this example a little bit to below?

```html
<tb-customization [customizableComponent]="customizableComponent">
  <span>Showing </span>
  <span>Default Text!</span>
</tb-customization>
```

I want to make sure this can properly support multiple child nodes.

Reply:
Thanks for the suggestion!